eng/mgmt/automation/utils.py

```diff
@@ -175,6 +178,19 @@ def update_service_ci_and_pom(sdk_root: str, service: str, group: str, module: s
             fout.write(pom_xml)
         logging.info("[POM][Success] Write to pom.xml")
 
+    if os.path.exists(changelog_file):
+        with open(changelog_file, "r") as fin:
+            changelog_str = fin.read()
+    logging.info("[CHANGELOG][Process] Adding initial section to changelog.md..")
+    initial_section = CHANGELOG_INITIAL_SECTION_FORMAT.format(artifact_id=module)
+    if initial_section not in changelog_str:
+        changelog_str += initial_section
+        with open(changelog_file, "w") as fout:
+            fout.write(changelog_str)
+        logging.info("[CHANGLOG][Success] Write to changelog.md.")
+    else:
+        logging.warning("[CHANGELOG] changelog.md already contains necessary sections. Will not add initial sections.")
```

Comment:
This seems only work for the 1st release.

Another candidate place is in `update_changelog` func.

Reply:
BTW, will it be invoked for SDK with existing releases, but without this `- Initial release for the {artifact_id} Java SDK.`?

Reply:
Yeah, this PR was intended only for 1st release.

Yes, there could be cases where there's no new features/breaking changes, though rare. 
Though I'm wondering what we should add to changelog content in this case. 
How about 
```md
### Other Changes

#### Dependency Updates

- Updated service `api-version`. 
```
?

Reply:
> BTW, will it be invoked for SDK with existing releases, but without this `- Initial release for the {artifact_id} Java SDK.`?

No, this method is only called when `require_sdk_integration` is true, which is 1st release.

Reply:
We can ignore this for now.

Reply:
> No, this method is only called when `require_sdk_integration` is true, which is 1st release.

Emm, found one exception in generating data-plane SDK if no readme:
https://github.com/Azure/azure-sdk-for-java/blob/20828f55b82becc3691550f90a9220fe77df9552/eng/mgmt/automation/generate_data.py#L346-L349

This is the only exception. I guess this still means 1st release?

Reply:
OK. DPG likely use a slightly different meaning of `require_sdk_integration`

Reply:
Then it should be OK, if only called for codegen of new lib.

Reply:
Updated the method's name to better reflex what it's meant to do.